src/config.rs

```diff
@@ -1964,15 +1961,6 @@ mod unified_read_pool_tests {
             ..cfg
         };
         invalid_cfg.validate().unwrap_err();
-        let invalid_cfg = UnifiedReadPoolConfig {
```

Comment:
The test case should be kept.

Reply:
Reverted, but also changed the expectation accordingly.